server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -207,20 +207,20 @@ def __init__(
         self._known_instances: dict[int, str] = known_instances or {}
         self._step_count = 0
         self._source_rows_subset = source_rows_subset
+        self.__id = uuid4().int
 
     def __init_subclass__(cls) -> None:
         ALL_TRANSLATORS[cls.DIALECT] = cls
 
-    @cache  # noqa: B019
```

Comment:
Why cas this cached? it doesn't see very costly ^^

Reply:
no idea :man_shrugging: